www/app/UtilityController.js

```diff
@@ -433,6 +433,8 @@ define(['app', 'livesocket'], function (app) {
 				else if ((item.Type == "Thermostat") && (item.SubType == "SetPoint")) {
 					status = "";
 					bigtext = item.Data + '\u00B0 ' + $scope.config.TempSign;
+					const regex = /ShowSetpointPopup\(event, \d*, \w*, (\d+\.?)*\d+\);/gm;
```

Comment:
One last thought.
What if signature for ShowSetpointPopup will change?
Shouldn't we make it depended on the JS code we have?

Reply:
> Do you also perform this action on the dashboard?

This proves my point, now I know about 3 (4?) different places where this is set.
- initialization of the web (dashboard)
- `refreshItem` (dashboard)
- During initialization of the web (Utils)
- `refreshItem` (Utils)

I feel like it should be wrapped into a class or some structure.
I would like to propose another PR, to parametrize this set point (I've already rose this in different [thread](https://github.com/domoticz/domoticz/issues/4055#issuecomment-993254561))

Step/limits would be useful.